src/test/java/com/homework18/dao/impls/ProjectMySqlDAOImplTest.java

```diff
@@ -0,0 +1,68 @@
+package com.homework18.dao.impls;
+
+import com.homework18.model.Project;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import java.util.Objects;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+
+public class ProjectMySqlDAOImplTest {
+    ApplicationContext context;
```

Comment:
What about encapsulation?
 ApplicationContext->ClassPathXmlApplicationContext in this case you don't need to use cast in the line 66.